src/main/java/uk/gov/hmcts/reform/civil/validation/UnavailableDateValidator.java

```diff
@@ -146,9 +146,9 @@ private boolean isSmallClaimHearingNullOrEmpty(SmallClaimHearing smallClaimHeari
         List<String> errors = new ArrayList<>();
 
         if (checkOneYearValidation(paymentDate)) {
-            errors.add("Date must be within the next 12 months");
+            errors.add("Date of first payment must be today or within the next 12 months");
         } else if (checkPastDateValidation(paymentDate)) {
-            errors.add("Date cannot be past date");
+            errors.add("Date of first payment must be today or within the next 12 months");
```

Comment:
since the error message is the same, you can collapse both ifs into a single if (.. or ..)

Reply:
good suggestion, change incorporated